tests/system/general/test_45_FilterSolutions/test_distributed_filter/test_distributed_topology.py

```diff
@@ -0,0 +1,541 @@
+        assert info.value.args[0] == "The First Segment Fat or Thin property is not supported for the selected topology"
+        design.topology.topology_list = TopologyList.STEPPED_IMPEDANCE
+        assert design.topology.first_fat
+        design.topology.first_fat = False
+        assert design.topology.first_fat is False
+        assert design.topology.circuit_response().splitlines() == read_resource_file("first_thin.ckt", "Distributed")
+
+    def test_distributed_use_series_caps(self):
+        design = ansys.aedt.core.filtersolutions.DistributedDesign(config["desktopVersion"])
+
+        with pytest.raises(RuntimeError) as info:
+            design.topology.use_series_caps = True
+        assert info.value.args[0] == "The Series Caps property is not supported for the selected topology"
+        design.attributes.filter_class = FilterClass.BAND_PASS
+        assert design.topology.use_series_caps is False
+        design.topology.use_series_caps = True
+        assert design.topology.use_series_caps
+        assert design.topology.circuit_response().splitlines() == read_resource_file(
+            "use_series_caps.ckt", "Distributed"
+        )
+
+    def test_distributed_combine_stubs(self):
+        design = ansys.aedt.core.filtersolutions.DistributedDesign(config["desktopVersion"])
+
+        with pytest.raises(RuntimeError) as info:
+            design.topology.combine_stubs = True
+        assert info.value.args[0] == "The Combine Stubs property is not supported for the selected topology"
+        design.attributes.filter_class = FilterClass.BAND_PASS
+        assert design.topology.combine_stubs is False
+        design.topology.combine_stubs = True
+        assert design.topology.combine_stubs
+        assert design.topology.circuit_response().splitlines() == read_resource_file("combine_stubs.ckt", "Distributed")
+
+    def test_use_coupled_lines(self):
+        design = ansys.aedt.core.filtersolutions.DistributedDesign(config["desktopVersion"])
+
+        with pytest.raises(RuntimeError) as info:
+            design.topology.use_coupled_lines = True
+        assert info.value.args[0] == "The Coupled Lines property is not supported for the selected topology"
+        design.attributes.filter_class = FilterClass.BAND_PASS
+        design.attributes.filter_type = FilterType.ELLIPTIC
+        assert design.topology.use_coupled_lines is False
+        design.topology.use_coupled_lines = True
+        assert design.topology.use_coupled_lines
+        assert design.topology.circuit_response().splitlines() == read_resource_file(
+            "use_coupled_lines.ckt", "Distributed"
+        )
+
+    def test_equal_width_approx(self):
+        design = ansys.aedt.core.filtersolutions.DistributedDesign(config["desktopVersion"])
+
+        with pytest.raises(RuntimeError) as info:
+            design.topology.equal_width_approx = True
+        assert info.value.args[0] == "The Equal Width property is not supported for the selected topology"
```

Comment:
Should this error message say something about class too? Or is it supported for all classes? 

This applies to a number of these tests. If we need to make a change to this test, we'll have to update the other tests impacted as well.